rotkehlchen/chain/evm/decoding/velodrome/velodrome_cache.py

```diff
@@ -322,37 +336,25 @@ def query_velodrome_like_data(
         cache_type: Literal[CacheType.VELODROME_POOL_ADDRESS, CacheType.AERODROME_POOL_ADDRESS],
         msg_aggregator: 'MessagesAggregator',
 ) -> list[VelodromePoolData] | None:
+    """Queries velodrome pools and tokens.
+    The actual onchain queries are run in a separate greenlet to avoid blocking decoding.
```

Comment:
shouldn't you change this comment now? -- also shouldn't you undo the logic for separate greenlet for curve too (and adjust any comments there too?)